### PR TITLE
i18n(zh-cn): Update install guide

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update install guide.
- Update `create astro`

### CODE REVIEW

File: src/content/docs/zh-cn/install/manual.mdx [hunk 1]
```diff
@@ -112,26 +144,49 @@ export default defineConfig({});
...
+<FileTree>
+- node_modules/
+- public/
+  - robots.txt
+- src/
+  - pages/
+    - index.astro
+  - env.d.ts
+- astro.config.mjs
+- package-lock.json or `yarn.lock`, `pnpm-lock.yaml`, etc.
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/install/manual.mdx:184]: This part (or `yarn.lock`, `pnpm-lock.yaml`, etc.) can be translated!

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @liruifengv, I added a small comment regarding something that wasn't translated, after addressing this, LGTM! 🚀

File: src/content/docs/zh-cn/install/manual.mdx (hunk 1 above)
Comment (liruifengv) [inline src/content/docs/zh-cn/install/manual.mdx:184]:
```suggestion
- package-lock.json 或者 `yarn.lock`，`pnpm-lock.yaml`，等等。
```

Comment (liruifengv) [reply src/content/docs/zh-cn/install/manual.mdx]: Thanks!

**Pull request merged**